fabfile.py

```diff
@@ -92,6 +95,18 @@ def deploy_cold():
     set_symlinks()
 
 
+def build_ui():
+    with lcd('ui'):
+        local('pwd')
+        local('npm install')
+        local('npm run-script bundle')
+
+    with lcd('front_end'):
+        local('npm install')
+        local('gulp build:dist')
```

Comment:
That's how TBG has set up the new app. We can change it later. I'd like to remove the gulp dependency.